### PR TITLE
Python eg

### PR DESCRIPTION
**Pull request by Artem-applitools**
Updates to run coverage generic tests on EG.

### CODE REVIEW

File: coverage-tests/configuration/eg_configuration.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
        "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    emitter: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js',
+    ],
+    template: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/template.hbs',
+    ext: '.py',
+    outPath: './test/coverage/generic',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    outPath: "./test/coverage/generic",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    name: "eyes_selenium_python",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js',
+    ],
+    template: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/template.hbs',
+    ext: '.py',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    ext: ".py",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js',
+    ],
+    template: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/template.hbs',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:8]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    template: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/template.hbs",
```

File: coverage-tests/configuration/eg_configuration.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js',
```
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js:6]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
        "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 4 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    name: "eyes_selenium_python",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 1 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
        "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 7 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
        "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/eg.overrides.js",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 2 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    emitter: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 5 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    ext: ".py",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 3 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    outPath: "./test/coverage/generic",
```

File: coverage-tests/configuration/eg_configuration.js (hunk 6 above)
Comment (Yarden-Ingber) [inline coverage-tests/configuration/eg_configuration.js]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
    template: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/template.hbs",
```

**Comment by Artem-applitools** [comment]
Requires updates for the python generic tests from PR https://github.com/applitools/sdk.coverage.tests/pull/102

**Review by ikseek — APPROVED** [review]
LGTM

**Pull request merged**